.github/workflows/release-arm.yaml

```diff
@@ -53,10 +53,9 @@ jobs:
           ./earthly.sh +all-arm --IMAGE_NAME=c3os-$FLAVOR-$TAG.img --IMAGE=quay.io/c3os/core-$FLAVOR:$TAG --MODEL=$MODEL --FLAVOR=$FLAVOR
           docker push quay.io/c3os/core-$FLAVOR:$TAG
       - name: Push  🔧
-        if: startsWith(github.ref, 'refs/tags/')
         env:
           FLAVOR: ${{ matrix.flavor }}
-          IMAGE: quay.io/c3os/c3os:${{ matrix.flavor }}-latest
+          IMAGE: quay.io/c3os/c3os-${{ matrix.flavor }}:latest
```

Comment:
I think this is not needed anymore there is a separate workflow pushing arm images now https://github.com/c3os-io/c3os/blob/b81e17ee026628a6cae70a6b0b1ff808145b99bf/.github/workflows/image-arm.yaml#L47